msi/make.cmd

```diff
@@ -0,0 +1,33 @@
+rmdir /s/q build
+
+rem TODO: Where should the git-credential-winstore, gcm and posh-git binaries
+rem       be found/stored?
```

Comment:
They should be bundled as Pacman packages first... But let's leave them out for the initial .msi support? We can worry about them once we have something working.